UnityProject/Assets/Scripts/Construction/Glass/glasssheet.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class glasssheet : MonoBehaviour
```

Comment:
Classes should be PascalCase. Change the name to GlassSheet. Btw is this class needed? It's empty

Reply:
humm, Since there's no good way of identifying components yet, currently doing full backup identifying by a certain component so for glass sheets there was no identifying component so I made one

Reply:
cool, just fix up the class name please and remove the un needed Start and Update hooks